index.html

```diff
@@ -1,55 +1,243 @@
+              <li class="about-list-item">Ruby on rails</li>
+              <li class="about-list-item">RSpec</li>
+              <li class="about-list-item">Capybara</li>
+              <li class="about-list-item">Selenium</li>
+            </ul>
+          </div>
+        </li>
+        <li class="about-box">
+          <div><img class="about-img" src="./Images/icon3.svg" alt="" /></div>
+          <div class="about-menu">
+            <h2 class="about-title">Skills</h2>
+            <ul class="about-list">
+              <li class="about-list-item">Codekit</li>
+              <li class="about-list-item">Github</li>
+              <li class="about-list-item">Codepen</li>
+              <li class="about-list-item">Gitlab</li>
+              <li class="about-list-item">Terminal</li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section class="fourth-section">
         <div class="container">
-            <h1 class="recent-header">My Recent Works</h1>
-            <hr>
+            <div class="contact">
+                <div><h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+                <form id="form" action="/">
+                    <div class="input-control">
+                        <input id="fullname" name="fullname" type="text" placeholder="Fullname">
+                        <div class="error"></div>
+                    </div>
+                    <div class="input-control">
+                        <input id="email" name="email" type="text" placeholder="Email address">
+                        <div class="error"></div>
+                    </div>
+                    <div class="input-control">
+                        <textarea  id="text" placeholder="Enter text here" ></textarea>
+                        <div class="error"></div>
+                    </div>
+                    <button class="box-button" type="submit">Get in touch</button>
+                    <ul class="social">
+                        <li>
+                          <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+                        </li>
+                        <li>
+                          <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+                        </li>
+                        <li>
+                          <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+                        </li>
+                        <li>
+                          <a href="#"><img src="./Images//github.svg" alt="" /></a>
+                        </li>
+                        <li>
+                          <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+                        </li>
+                      </ul>
+                      <h4>danielutibe07@gmail.com</h5>
+                </form>
+            </div>
         </div>
-    </section> -->
+    </section>
     <script src="app.js"></script>
```

Comment:
- [ ] [OPTIONAL] To allow you to move faster, I suggest you remove these lines since you are not working on them for now. This will reduce your potential bugs and allow you to focus on important things.